emails/file-expiry.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+    Hr,
+    Link,
+    Section,
+    Text
+} from "@react-email/components";
+import { z } from "zod";
+import { Email } from "./_common/watcloud-email";
+
+const EmailProps = z.object({
+    name: z.string(),
+    paths: z.array(z.string()),
+    daysForExpiry: z.string(),
+    deletionDate: z.string()
+});
+
+type EmailProps = z.infer<typeof EmailProps>;
+
+export const FileExpiryEmail = (props: EmailProps) => {
+    const { name, paths, daysForExpiry, deletionDate} = EmailProps.parse(props);
+
+    const previewText = `You have expired files in the WATO drives`;
+    const mapPathToMachine = (): Record<string, string[]> => {
```

Comment:
I feel like we can just keep this simple and just list the strings, don't need to group by machine. But that's up to you.